File: backend/app/schemas/odds.py
from datetime import datetime

from pydantic import BaseModel, ConfigDict


class OddsOut(BaseModel):
    model_config = ConfigDict(from_attributes=True)

    id: int
    fixture_id: int
    casa_de_aposta: str
    mercado: str
    selecao: str
    odd: float
    prob_implicita: float
    timestamp: datetime

File: frontend/src/types/odds.ts
export interface Odds {
  id: number;
  fixture_id: number;
  casa_de_aposta: string;
  mercado: string;
  selecao: string;
  odd: number;
  prob_implicita: number;
  timestamp: string;
}

export interface League {
  id: number;
  nome: string;
  pais: string;
  temporada: number;
}
